.circleci/config.yml

```diff
@@ -167,7 +189,7 @@ workflows:
                 - master
       - deploy_s3:
           context: wpe-product-info-aws
-          environment: staging/themes
+          # environment: staging/themes
```

Comment:
Should this be removed rather than commented out?

Reply:
We will be un-commenting it after the initial deploy to S3